src/pages/settings/Profile/ProfileAvatar.tsx

```diff
@@ -38,7 +38,7 @@ function ProfileAvatar({route}: ProfileAvatarProps) {
             defaultOpen
             source={getFullSizeAvatar(avatarURL, accountID)}
             onModalClose={() => {
-                Navigation.goBack();
+                Navigation.goBack(route.params?.backTo ?? undefined);
```

Comment:
```suggestion
                Navigation.goBack(route.params?.backTo);
```

Would this produce a different behavior?